app/templates/components/preprint-form-authors.hbs

```diff
@@ -114,12 +121,12 @@
                                 {{#if contrib.unregisteredContributor}}
                                     {{contrib.unregisteredContributor}}
                                 {{else}}
-                                    <a href={{contrib.users.profileUrl}}> {{contrib.users.fullName}} </a>
+                                    <a href={{contrib.users.links.html}}> {{contrib.users.fullName}} </a>
```

Comment:
It just wasn't working. I do not know why. When the page loaded, contrib.users.profileUrl was undefined, so none of the profile links worked.